webui/react/src/utils/time.ts

```diff
@@ -0,0 +1,29 @@
+// @ts-ignore
+import humanizeDuration from 'humanize-duration';
+
+import { Experiment } from 'types';
+
+// Experiment duration (naive) in miliseconds
+export const experimentDuration = (experiment: Experiment): number => {
+  const endTime = experiment.endTime ? new Date(experiment.endTime) : new Date();
+  const startTime = new Date(experiment.startTime);
+  return endTime.getTime() - startTime.getTime();
+};
+
+export const shortEnglishHumannizer = humanizeDuration.humanizer({ language: 'shortEn',
```

Comment:
nit: newline between `{` and `language: 'shortEn',`?

Reply:
the auto sort keys fixed this :tada: